modules/parent-join/build.gradle

```diff
@@ -16,7 +16,7 @@ esplugin {
 
 restResources {
   restApi {
-    include '_common', 'bulk', 'cluster', 'nodes', 'indices', 'index', 'search'
+    include '_common', 'bulk', 'cluster', 'get', 'nodes', 'indices', 'index', 'search'
```

Comment:
Why is this change needed? The new yaml tests does do get api requests, but that is in a different module then this change is.